test/withdrawUnitTest.js

```diff
@@ -25,10 +25,10 @@ contract('ROSCA withdraw Unit Test', function(accounts) {
 
     it("Throws when calling withdraw from a non-member", co(function* () {
         yield Promise.all([
-            ethRoscaHelper.contribute(0, consts.CONTRIBUTION_SIZE),
-            ethRoscaHelper.contribute(1, consts.CONTRIBUTION_SIZE),
-            ethRoscaHelper.contribute(2, consts.CONTRIBUTION_SIZE),
-            ethRoscaHelper.contribute(3, consts.CONTRIBUTION_SIZE),
+            ethRoscaHelper.contribute(0, 2 * consts.CONTRIBUTION_SIZE),
```

Comment:
this is not related to Round 0 -> 1?

Reply:
it is, this test originally test withdrawal in round 0 , but we are in round 1 , we need to contribute more than round 1's contribution to be able to withdraw